packages/pwa-kit-runtime/src/utils/morgan-stream.js

```diff
@@ -1,20 +0,0 @@
-/*
- * Copyright (c) 2022, salesforce.com, inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-
-import {Writable} from 'stream'
-/**
- * @private
- * A stream that uses console.log to write log lines
- */
-class ConsoleLogStream extends Writable {
-    write(line) {
-        console.log(line)
-    }
-}
-// create new stream to be used in morgan stream
-let morganStream = new ConsoleLogStream()
-export {morganStream}
```

Comment:
Breaking?

Reply:
Removing the `morganStream` could be seen as breaking. We can leave it and mark it as deprecated and replace the `console.log` with `logger.info`.

Reply:
Sounds good.. We should probably create a ticket to remove morganStream in the upcoming major version.